client/src/types/general.ts

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+/* eslint-disable no-unused-vars */
+export enum UserRole {
+  User = 'USER',
+  Admin = 'ADMIN'
+}
```

Comment:
This works for this PR, but I would like to have a single code module for types shared by the frontend and backend, so we don't need to manually keep them in sync or duplicate code between `server` and `client`.

Reply:
So something like folder "common" to the project root and subfolder "types" where we place client/server shared types?

Reply:
So somehow with declaring global types and no need for importing. I will leave this for now as is, let's think this one later.